flask_lesson_intro/utils.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+
+def get_data():
+    with open("data.json") as file:
+        return json.load(file)
```

Comment:
what if the file will not found or contains not valid data

Reply:
FileNotFoundError: [Errno 2] No such file or directory: 'data.json' 
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 14 column 3 (char 2063)
– we can see one of these errors )

Reply:
Great, maybe you should catch them?